bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +23,280 @@ class CoffeeLintBear:
     CAN_DETECT = {'Syntax', 'Formatting', 'Smell'}
 
     severity_map = {'warn': RESULT_SEVERITY.NORMAL,
-                    'error': RESULT_SEVERITY.MAJOR}
+                    'error': RESULT_SEVERITY.MAJOR,
+                    'ignore': RESULT_SEVERITY.INFO}
 
     @staticmethod
     def create_arguments(filename, file, config_file):
-        return '--reporter=raw', filename
+        return '--reporter=raw', '-f', config_file, filename
+
+    @staticmethod
+    def generate_config(filename, file,
+                        max_line_length: int=79,
+                        max_line_length_affect_comments: bool=True,
+                        space_before_and_after_arrow: bool=True,
+                        check_braces_spacing: bool=False,
+                        braces_spacing_width: int=1,
+                        spacing_in_empty_braces: int=1,
+                        class_naming_camel_case: bool=True,
+                        spaces_before_colon: int=-1,
+                        spaces_after_colon: int=-1,
+                        enforce_newline_at_EOF: bool=True,
+                        use_spaces: bool=True,
+                        tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
```

Comment:
What is ``tab_width`` then? :D
On 14 Jul 2016 10:37 pm, "Asnelchristian" <notifications@github.com> wrote:

In bears/coffee_script/CoffeeLintBear.py
<https://github.com/coala-analyzer/coala-bears/pull/555#discussion_r70843840>
:

> +        return '--reporter=raw', '-f', config_file, filename
> +
> +    @staticmethod
> +    def generate_config(filename, file,
> +                        max_line_length: int=79,
> +                        max_line_length_affect_comments: bool=True,
> +                        space_before_and_after_arrow: bool=True,
> +                        check_braces_spacing: bool=False,
> +                        braces_spacing_width: int=1,
> +                        spacing_in_empty_braces: int=1,
> +                        class_naming_camel_case: bool=True,
> +                        spaces_before_colon: int=-1,
> +                        spaces_after_colon: int=-1,
> +                        enforce_newline_at_EOF: bool=True,
> +                        use_spaces: bool=True,
> +                        tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,

when use_spaces is true the value for indentation is set to true then 2
spaces are used

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/coala-analyzer/coala-bears/pull/555/files/f0b436c93032c451453f0a650f4c8c342e427fa3#r70843840>,
or mute the thread
<https://github.com/notifications/unsubscribe/AHDgOXZx4I-8zGU_B9Ek8CmK53bdqt9-ks5qVmzqgaJpZM4I9f6t>
.